lib/mongo/index/view.rb

```diff
@@ -78,25 +79,30 @@ class View
       #   view.drop_one('name_1')
       #
       # @param [ String ] name The name of the index.
+      # @param [ String, Bson::Document, Hash ] comment A user-provided
+      #   comment to attach to this command.
       #
       # @return [ Result ] The response.
       #
       # @since 2.0.0
-      def drop_one(name)
+      def drop_one(name, comment: nil)
         raise Error::MultiIndexDrop.new if name == Index::ALL
-        drop_by_name(name)
+        drop_by_name(name, comment: comment)
       end
 
       # Drop all indexes on the collection.
       #
       # @example Drop all indexes on the collection.
       #   view.drop_all
       #
+      # @param [ String, Bson::Document, Hash ] comment A user-provided
+      #   comment to attach to this command.
+      #
       # @return [ Result ] The response.
       #
       # @since 2.0.0
-      def drop_all
-        drop_by_name(Index::ALL)
+      def drop_all(comment: nil)
```

Comment:
I think explicitly specifying options like this is problematic because it is not forward-compatible, i.e. if a future driver version adds another option, application cannot specify that option when running on an older driver. I would use '**options` or `options` as we do elsewhere.

Reply:
Thanks, changed to `options`.